src/app/components/PropertyMeta.js

```diff
@@ -1,92 +1,170 @@
-import {Input,InputLabel,FormControl,Select,MenuItem} from '@material-ui/core';
-import {notifyDataUpdate} from '../util/util';
-import {useEffect, useState} from 'react';
-import {viewAlbum} from '../uploads/core-sdk';
-const Photos = (props)=>{
-  console.log('photos',props._photos);
-  const pics=props._photos&&props._photos.map((photo)=>{
-    return <img src={photo}/>;
-  });
-  return pics;
-}
-export const PropertyMeta = ()=> {
-    useEffect(()=>{
-      listFiles();
+import {
+  Input,
+  InputLabel,
+  FormControl,
+  Select,
+  MenuItem,
+} from "@material-ui/core";
+import { notifyDataUpdate } from "../util/util";
+import { useEffect, useState } from "react";
+import { viewAlbum } from "../uploads/core-sdk";
+const Photos = (props) => {
+  console.log("photos", props._photos);
+  const pics =
+    props._photos &&
+    props._photos.map((photo) => {
+      return <img src={photo} />;
     });
```

Comment:
Nothing much changed except formatting till this point

Reply:
This is the auto-formatted by VS code.